src/main.rs

```diff
@@ -53,6 +53,12 @@ fn main() {
     let in_dir = options.input_dir;
     let out_dir = options.output_dir;
 
+    if !in_dir.exists() {
+        panic!("No such file or directory {:?}", in_dir);
```

Comment:
Could you make this a eprintln!() with an std::process::exit(1) after? panic always looks like a bug :)